src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,44 @@
 import React from 'react';
+import './PostInfo.scss';
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+interface PostInfoProps {
+  postInfo: Post
+}
+
+export const PostInfo: React.FC<PostInfoProps> = ({
+  postInfo: {
+    title, body, user, comments,
+  },
+}) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        {user ? <UserInfo userInfo={user} /> : undefined}
+      </p>
+    </div>
+
+    <p className="PostInfo__body">
+      {body}
+    </p>
+
+    { comments
```

Comment:
check for `comment.length`. It's the only way to check if an array is truthy or falsy in the way you want it